src/testdir/test_termdebug.vim

```diff
@@ -152,6 +152,71 @@ func Test_termdebug_basic()
   %bw!
 endfunc
 
+func Test_termdebug_tbreak()
+  " the same as the first half part of Test_termdebug_basic(),
+  " but only 'debugPC' sign should be present after the breakpoint
+  " was hit once
+  let lines =<< trim END
+    #include <stdio.h>
+    #include <stdlib.h>
+
+    int isprime(int n)
+    {
+      if (n <= 1)
+        return 0;
+
+      for (int i = 2; i <= n / 2; i++)
+        if (n % i == 0)
+          return 0;
+
+      return 1;
+    }
+
+    int main(int argc, char *argv[])
+    {
+      int n = 7;
+
+      printf("%d is %s prime\n", n, isprime(n) ? "a" : "not a");
+
+      return 0;
+    }
+  END
+  call writefile(lines, 'XTD_basic.c', 'D')
+  call system($'{g:GCC} -g -o XTD_basic XTD_basic.c')
+
+  edit XTD_basic.c
```

Comment:
A different file name should be used.  The "XTD_basic.c" is used in Test_termdebug_basic().